server/index.js

```diff
@@ -227,46 +231,57 @@ app.post('/api/likes', (req, res, next) => {
 
 app.get('/api/likes/:postId', (req, res, next) => {
   const postId = Number(req.params.postId);
-  const firstSql = `
+  const sql = `
     select count("l".*) as "totalLikes"
     from "likePosts" as "l"
     where "postId" = $1
     `;
-  const secondSql = `
+
+  const params = [postId];
+
+  db.query(sql, params)
+    .then(firstResult => {
+      if (!firstResult.rows) {
+        throw new ClientError(400, `cannot find post with postId ${postId}`);
+      }
+      const [totalLikes] = firstResult.rows;
```

Comment:
`.rows` will always be truthy (even empty arrays are truthy), so this condition will never run. However, if the first element of said array is undefined, then you cannot find the post. But that status code would be a `404`. See if you can find/fix that anywhere else in your code.

Reply:
Resolved, thanks for the feedback!